CHANGELOG/v1.5.1.md

```diff
@@ -0,0 +1,50 @@
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.24.x -> v1.28.x
+- Workload Cluster: v1.22.x -> v1.28.x
+
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Changes since v1.5.0
+---
+## :chart_with_upwards_trend: Overview
+- 33 new commits merged
+- 2 feature additions ✨
+- 9 bugs fixed 🐛
+
+## :sparkles: New Features
+- MachineSet: Adjust preflight check to allow kubelet version skew of 3 for clusters running v1.28 and above (#9233)
+- Testing/documentation: V1.28: Prepare quickstart, capd and tests for the new release includi… (#9225)
```

Comment:
```suggestion
- Testing/documentation: v1.28: Prepare quickstart, capd and tests for the new release including kind bump (#9225)
```
nit

Reply:
hmm, good catch that is due to long PR title + cherry pick PR prefix added making this